src/styles/variables.sass

```diff
@@ -0,0 +1,49 @@
+// Favorite constants
+$gratio: 1.618
+$pi: 3.14159
+
+// Dimensions
+$radius: 2px
+$spacing: 24px
+$padding: 10px
+
+// Font properties
+$font-family-sans-serif: 'Helvetica Neue', Helvetica, Arial, sans-serif
+$font-family-serif: Georgia, Cambria, 'Times New Roman', Times, serif
+$font-family-monospace: Monaco, Menlo, Consolas, 'Courier New', monospace
+
+$font-family-base: $font-family-sans-serif
+$font-family-head: $font-family-serif
+
+$base-font-size: 16px
+$fw-normal: 400
+$fw-medium: 500
+$fw-semibold: 600
+$fw-bold: 700
```

Comment:
Je to v zásadě totéž, jako jsem [vysvětloval](https://github.com/manGoweb/mango-cli-example/commit/8f5d1a9514e08184a858bcff8a527e72d7fbed7f#commitcomment-21192102) u media queries ‒ takhle Ti to zařve při překlepu, líp Ti našeptává IDE, nemusíš si přesně pamatovat čísla a změny jsou snáz proveditelné. Ani jednu z těchto výhod psaní konstant nemá.